packages/integration-tests/schema/enums.graphql

```diff
@@ -0,0 +1,2 @@
+enum PublisherSize { SMALL LARGE }
```

Comment:
In theory prettier supports graphql, so I'm assuming this would get prettier'd in the downstream project. Didn't bother with that for now/here.